### PR TITLE
feat: column resize for virtual scroll

### PR DESCRIPTION
**Pull request by haxxmaxx**

### CODE REVIEW

File: src/table/virtualized-table/TotalsCell.tsx [hunk 1]
```diff
@@ -31,7 +34,10 @@ const TotalsCell = ({ index, style, data }: TotalsCellProps) => {
         display: 'flex',
         alignItems: 'center',
         borderStyle: 'solid',
-        borderWidth: isLastColumn ? '0px' : '0px 1px 0px 0px',
+        borderLeftWidth: '0px',
```
Comment (haxxmaxx) [inline src/table/virtualized-table/TotalsCell.tsx:37]: For some reason there seems to be a 3px border by default or inherited from somewhere. I couldn't see it anywhere in the devtools except under the layout tab

Comment (cbt1) [reply src/table/virtualized-table/TotalsCell.tsx:37]: Yeah I've noticed that as well, could be some default browser behaviour

File: src/table/virtualized-table/__tests__/Table.spec.tsx [hunk 2]
```diff
@@ -100,6 +103,10 @@ describe('<Table />', () => {
     isModal = false;
 
     constraints = {};
+
+    rootElement = {
+      getBoundingClientRect: () => ({ height: rect.width }),
```
Comment (cbt1) [inline src/table/virtualized-table/__tests__/Table.spec.tsx:108]: A bit confusing have the height equal the width. Why not do set it from `rect.height`?

Comment (haxxmaxx) [reply src/table/virtualized-table/__tests__/Table.spec.tsx]: suppose to be height, fixed

**Review by cbt1 — APPROVED** [review]

**Pull request merged**